src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    /**
+     * Global variables
+     */
+    private static final String SPACE = "    ";
+    private static final String LINE = SPACE + "______________________________________________________________________";
+    private static final String END_LINE = SPACE + "======================================================================\n";
+    private static final String LOGO = "\n" +
+            "███████████████████████████████\n" +
+            "█▄─▄─▀█─█─█▄─██─▄█─▄─▄─█▄─██─▄█\n" +
+            "██─▄─▀█─▄─██─██─████─████─██─██\n" +
+            "▀▄▄▄▄▀▀▄▀▄▀▀▄▄▄▄▀▀▀▄▄▄▀▀▀▄▄▄▄▀▀";
+    private Items items;
+
+    public Ui() {
+        items = new Items();
+    }
+
+    /**
+     * returns a greeting to the user
+     */
+    public void greet() {
+        System.out.println(LOGO);
+        printMessage("Hello! I'm Bhutu, your personal chatbot!\nWhat can I do for you?");
+    }
+
+    /**
+     * Gets the user input
```

Comment:
Perhaps need an empty line between description and parameter section? 🤔
And I noticed the same issue in several other places below.